.gitignore

```diff
@@ -35,3 +35,5 @@ jspm_packages/
 .env
 .env.test
 .env.local
+.env_local
```

Comment:
Will you rename your `.env_local` file to match the `.env.local` format so we don't need this deviation in file names.

Reply:
Also, should use dots (.) on the elephantsql file.

Is this even needed if you put the vars into `.env.local` since it only applies to you.

Reply:
I am using .env for my environmental variables, but I wanted to be able to use ElephantSQL too, so I created several, .env files. But the one being used is .env. The others are added to gitignore to make sure that secrets hidden in those files don't get pushed to Github

Reply:
You can put alternative vars in the same `.env` but comment them out with a `#`.

If you want to have multiple env files then perhaps having `.env*` in git ignore would be more appropriate.